docs/guides/platform/get-started/getting-started-after-deployment/index.md

```diff
@@ -0,0 +1,244 @@
+
+### CentOS/RHEL Stream and Fedora
+
+*This includes CentOS Stream (and 8), other RHEL derivatives (including AlmaLinux 8, and Rocky Linux 8), and Fedora.*
+
+    dnf upgrade
+
+### Ubuntu and Debian
+
+    apt update && apt upgrade
+
+{{< note >}}
+You may be prompted to make a menu selection when the Grub package is updated on Ubuntu. If prompted, select `keep the local version currently installed`.
+{{< /note >}}
+
+### Other Distributions
+
+#### Alpine
+
+    apk update && apk upgrade
+
+#### Arch Linux
+
+    pacman -Syu
+
+#### CentOS 7
+
+    yum update
+
+#### Gentoo
+
+    emaint sync -a
+
+After running a sync, it may end with a message that you should upgrade Portage using a `--oneshot` emerge command. If so, run the Portage update. Then update the rest of the system:
+
+    emerge -uDU --keep-going --with-bdeps=y @world
+
+#### OpenSUSE
+
+    zypper update
+
+#### Slackware
+
+    slackpkg update
+    slackpkg upgrade-all
+
+## Set the Hostname
+
+A hostname is used to identify your Linode using an easy-to-remember name. Your Linode's hostname doesn't necessarily associate with websites or email services hosted on the system, but see our guide on using the [hosts file](/docs/networking/dns/using-your-systems-hosts-file/) if you want to assign your Linode a fully qualified domain name.
+
+ Your hostname should be something unique, and should not be *www* or anything too generic. Some people name their servers after planets, philosophers, or animals. After you've made the change below, you may need to log out and log back in again to see the terminal prompt change from `localhost` to your new hostname. The command `hostname` should also show it correctly.
+
+### Most Distributions
+
+*This includes Ubuntu 16.04 (and newer), CentOS 7 (and newer), other RHEL derivatives (including AlmaLinux 8 and Rocky Linux 8), Debian 8 (and newer), Fedora, OpenSuse, and Arch.*
+
+Replace `example-hostname` with one of your choice.
+
+    hostnamectl set-hostname example-hostname
+
+### Other Distributions
+
+#### Alpine
+
+See [Update Your Systems hosts File](#update-your-systems-hosts-file).
+
+#### Gentoo
+
+    echo "HOSTNAME=\"example-hostname\"" > /etc/conf.d/hostname
+    /etc/init.d/hostname restart
+
+#### Slackware
+
+    echo "example-hostname" > /etc/HOSTNAME
+    hostname -F /etc/HOSTNAME
+
+### Update Your System's `hosts` File
+
+The `hosts` file creates static associations between IP addresses and hostnames or domains which the system prioritizes before DNS for name resolution. Open this file in a text editor and add a line for your Linode's public IP address. You can associate this address with your Linode's **Fully Qualified Domain Name** (FQDN) if you have one, and with the local hostname you set in the steps above. In the example below, `203.0.113.10` is the public IP address, `example-hostname` is the local hostname, and `example-hostname.example.com` is the FQDN.
```

Comment:
Should we include mention of how to open a file in a text editor, given the target demographic?

Reply:
Great callout. Just made a commit to address this.